src/shared_styles/general.scss

```diff
@@ -7,6 +7,7 @@ ul, ol {
 body {
   color: $gray-darkest;
   font-family: $standard-font;
+  --green: #{$green};
```

Comment:
On second thought, it might make sense to have all colors articulated here in general as opposed to typography